tests/examples/test_basic.py

```diff
@@ -25,6 +25,6 @@ def test_deploy_settings(sett_id):
 
     assert strategy.governance() == BADGER_DEV_MULTISIG
 
-    assert strategy.performanceFeeGovernance() == config.params.performanceFeeGovernance
+    assert strategy.performanceFeeGovernance() == 2000
```

Comment:
can i ask why you hardcode here and not adjust config?

Reply:
I decided to to adjust the parameters just yet because that would deploy the strats on fork test using the new fee instead of changing it, so I was planning on updating the config in a separate PR after executing the change. But now that I think about it, it wouldn't really have impacted the output of the tests. Will make sure to adjust this assertion when I fix the config. Thanks!